ios/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-platform :ios, '11.0'
+platform :ios, '12.0'
```

Comment:
Is this a required change?

Reply:
Minimum iOS version for latest Flutter is 12 [official doc](https://docs.flutter.dev/deployment/ios)

Reply:
Most older iPhones run 12+ iOS